src/common/playlists/SignInMethods.jsx

```diff
@@ -0,0 +1,36 @@
+import { Typography, Box, Stack, Button } from "@mui/material";
```

Comment:
Even though, i saw the comment however i would still ask to remove it.

Reply:
removed